### PR TITLE
INTYGFV-14465: Implemented getting drafts as list

### PR DESCRIPTION
**Pull request by mhornfeldt**

### CODE REVIEW

**Comment by mhornfeldt** [comment]
Not ready for review

File: web/src/main/java/se/inera/intyg/webcert/web/service/facade/list/ListDecorator.java
Comment (mww-sopra-steria) [inline web/src/main/java/se/inera/intyg/webcert/web/service/facade/list/ListDecorator.java:27]: Is it really decoration when it returns a new list? Do you need to have a return value because the method is changing the items that are sent in as parameters.

Comment (mww-sopra-steria) [reply web/src/main/java/se/inera/intyg/webcert/web/service/facade/list/ListDecorator.java:27]: You can simplify the implementation if you change this.

File: web/src/main/java/se/inera/intyg/webcert/web/service/facade/list/ListDraftsFacadeServiceImpl.java
Comment (mww-sopra-steria) [inline web/src/main/java/se/inera/intyg/webcert/web/service/facade/list/ListDraftsFacadeServiceImpl.java:82]: If you change the decorator implementation you can modify this as well.

File: web/src/main/java/se/inera/intyg/webcert/web/service/facade/list/config/GetStaffInfoFacadeServiceImpl.java
Comment (mww-sopra-steria) [inline web/src/main/java/se/inera/intyg/webcert/web/service/facade/list/config/GetStaffInfoFacadeServiceImpl.java:63]: Here is some authentication logic hidden in this class. Can this be moved?

**Review by mww-sopra-steria — CHANGES_REQUESTED** [review]
Overall looks very good! I have a few comments that you should take a look at before merging.

**Review by mww-sopra-steria — APPROVED** [review]
That was quick! It is easy to refactor with a good test suite! Well done! :-)

**Pull request merged**
